### ISSUE TITLE
Input/Field Grouping

### ISSUE THREAD

**Issue by benhaynes** [directus/app#717]
#### :rotating_light: MIGRATED FROM REQUESTS.GETDIRECTUS.COM :rotating_light: 

# Input/Field Grouping
### :+1: = 7
*Created 1 year ago by @benhaynes*
 

-----
### @benhaynes – (1 year ago)
There's front-end code to support grouping fields into sets of inputs (similar to how "active" is in it's own group). Each grouping would have a title and separates the inputs into a new white box container.

For features like this we should first look at the best way to save the data in the database, then we can find a suitable interface to make it easier to change. Since developers will be the primary people setting up the framework, we can assume they're comfortable doing some setup directly in the database... and we can wait on the Settings interfaces.

I imagine this could be saved within directus_columns either as it's own record "delimiting" a space between groups (using the sort column), or as a new "group" column in that table that stores the parent group for that input.

As for an interface, on the Settings > Tables & Inputs there should be a "Add New Grouping Label". Clicking that adds a row where the user can type a label.

@benhaynes to design

 

-----
### @davidbwaters – (1 year ago)
I'd like to see this with an option for repeatable field groups, similar to what Prismic and similar have for more flexibility in organizing content without seperate tables for everything. I have no idea how easy this would be to impliment

**Comment by rijkvanzanten** [comment]
Duplicate of #773
